compiler/src/org.graalvm.compiler.asm.aarch64/src/org/graalvm/compiler/asm/aarch64/AArch64Assembler.java

```diff
@@ -368,42 +368,16 @@ private static int rn(Register reg) {
         return reg.encoding << RnOffset;
     }
 
-    /**
-     * Enumeration of all different lane types of SIMD register.
-     *
-     * Byte(B):8b/lane; HalfWord(H):16b/lane; Word(S):32b/lane; DoubleWord(D):64b/lane.
-     */
-    public enum SIMDElementSize {
-        Byte(0, 8),
-        HalfWord(1, 16),
-        Word(2, 32),
-        DoubleWord(3, 64);
-
-        public final int encoding;
-        public final int nbits;
-
-        SIMDElementSize(int encoding, int nbits) {
-            this.encoding = encoding;
-            this.nbits = nbits;
-        }
-    }
-
     /**
      * Enumeration of all different instruction kinds: General32/64 are the general instructions
      * (integer, branch, etc.), for 32-, respectively 64-bit operands. FP32/64 is the encoding for
-     * the 32/64bit float operations. SIMDByte/HalfWord/Word/DoubleWord is the encoding for SIMD
-     * instructions
+     * the 32/64bit float operations.
      */
     protected enum InstructionType {
         General32(0b00 << 30, 32, true),
```

Comment:
I detect that this code is problematic. According to the [Bad practice (BAD_PRACTICE)](https://spotbugs.readthedocs.io/en/stable/bugDescriptions.html#bad-practice-bad-practice), [Nm: Method names should start with a lower case letter (NM_METHOD_NAMING_CONVENTION)](https://spotbugs.readthedocs.io/en/stable/bugDescriptions.html#nm-method-names-should-start-with-a-lower-case-letter-nm-method-naming-convention).
Methods should be verbs, in mixed case with the first letter lowercase, with the first letter of each internal word capitalized.